runbotics/runbotics-orchestrator-ui/src/src-app/views/users/UsersListView/UsersListEdit/UsersListEditForm.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { FC, ChangeEvent } from 'react';
+
+import { TextField, MenuItem } from '@mui/material';
+
+import useTranslations from '#src-app/hooks/useTranslations';
+import { emailFieldValidator, requiredFieldValidator } from '#src-app/views/users/UsersListView/UsersListEdit/UsersListEdit.utils';
+
+import { UsersListEditFormProps } from './UsersListEdit.types';
+
+const UsersListEditForm: FC<UsersListEditFormProps> = ({
+    user,
+    setUser,
+    validation,
+    setValidation
+}) => {
+    const emailRegex = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,})+$/;
+
+    const { translate } = useTranslations();
+
+    const getEmailValidator = () => !(validation.email) && emailFieldValidator;
+
+    const getLoginValidator = () => !(validation.login) && requiredFieldValidator;
+
+    const handleEmailChange = (event: ChangeEvent<HTMLInputElement>) => {
+        const email = event.target.value;
+        setUser((prevState) => ({ ...prevState, email }));
+        setValidation((prevState) => ({
+            ...prevState,
+            email: (email.trim() !== '') && emailRegex.test(email)
+        }));
+    };
+
+    const handleLoginChange = (event: ChangeEvent<HTMLInputElement>) => {
+        const login = event.target.value;
+        setUser((prevState) => ({ ...prevState, login }));
+        setValidation((prevState) => ({ ...prevState, login: (login.trim() !== '' ) }));
+    };
+
+    const handleFirstNameChange = (event: ChangeEvent<HTMLInputElement>) => {
+        setUser((prevState) => ({ ...prevState, firstName: event.target.value}));
+    };
+
+    const handleLastNameChange = (event: ChangeEvent<HTMLInputElement>) => {
+        setUser((prevState) => ({ ...prevState, lastName: event.target.value }));
+    };
+
+    const handleLanguageChange = (event: ChangeEvent<HTMLInputElement>) => {
```

Comment:
I've been told to use "handle" keyword in cases when it's hard to describe what function does - here it's easy:  so name it changeUserLanguage 

same with handleLastNameChange, handleFirstNameChange and handleLoginChange

handleEmailChange is more complicated, so u can keep it